modules/machineset-yaml-rhv.adoc

```diff
@@ -0,0 +1,108 @@
+    machine.openshift.io/cluster-api-machine-role: <role> <2>
+    machine.openshift.io/cluster-api-machine-type: <role> <2>
+  name: <infrastructure_id>-<role> <3>
+  namespace: openshift-machine-api
+spec:
+  replicas: <number_of_replicas> <4>
+  Selector: <5>
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+        machine.openshift.io/cluster-api-machine-role: <role> <2>
+        machine.openshift.io/cluster-api-machine-type: <role> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role> <3>
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/<role>: "" <2>
+      providerSpec:
+        value:
+          apiVersion: ovirtproviderconfig.machine.openshift.io/v1beta1
+          cluster_id: <ovirt_cluster_id> <6>
+          template_name: <ovirt_template_name> <7>
+          instance_type_id: <instance_type_id> <8>
+          cpu: <9>
+            cores: <number_of_cores> <10>
+            sockets: <number_of_sockets> <11>
+            threads: <number_of_threads> <12>
+          memory_mb: <memory_size> <13>
+          os_disk: <14>
+            size_gb: <disk_size> <15>
+          network_interfaces: <16>
+            vnic_profile_id:  <vnic_profile_id> <17>
+          credentialsSecret:
+            name: ovirt-credentials <18>
+          kind: OvirtMachineProviderSpec
+          type: <workload_type> <19>
+          userDataSecret:
+            name: worker-user-data
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI (`oc`) installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+
+<2> Specify the node label to add.
+
+<3> Specify the infrastructure ID and node label.
+
+<4> Specify the number of machines to create.
+
+<5> Selector for the machines.
+
+<6> Specify the UUID for the {rh-virtualization} cluster to which this VM insce belongs.
+
+<7> Specify the {rh-virtualization} VM template to use to create the machine.
+
+<8> (Optional) Specify the VM instance type. If you include this parameter, you do not need to specify the hardware parameters of the VM including CPU and memory because this parameter overrides all hardware parameters.
+
+<9> (Optional) The CPU field contains the CPU's configuration, including sockets, cores, and threads.
+
+<10> (Optional) Specify the number of sockets for a VM.
+
+<11> (Optional) Specify the number of cores per socket.
+
+<12> (Optional) Specify the number of threads per core.
+
+<13> (Optional) Specify the size of a VM's memory in MiBs.
```

Comment:
MBs instead of MiBs like in RHV UI
![image](https://user-images.githubusercontent.com/73419853/106011166-de093780-60c2-11eb-8de5-6e662a0ab4d9.png)

Reply:
Its Mib:
![image](https://user-images.githubusercontent.com/38602062/106142722-80d0bd00-617a-11eb-9876-a8a0c57ef558.png)
and our code:
![image](https://user-images.githubusercontent.com/38602062/106142816-a1991280-617a-11eb-94c1-11fb7259cb9c.png)

Reply:
Everything should be aligned with RHV.
For 7 years the size in RHV for VMs memory is in MB and GB.